pom.xml

```diff
@@ -1663,6 +1663,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
           <version>${proto-backwards-compatibility.version}</version>
           <configuration>
             <protoSourceRoot>${basedir}/target/classes</protoSourceRoot>
+            <allowBreakingChanges>true</allowBreakingChanges>
```

Comment:
I don't think we should add this option.  The whole point of backwards compatibility check is to prevent breaking changes.

Reply:
got it! sure, will revert it back~